packages/cli/internal/pkg/cli/actionable/error.go

```diff
@@ -0,0 +1,38 @@
+package actionable
+
+import (
+	"fmt"
+	"strings"
+)
+
+type Error struct {
+	Cause           error
+	SuggestedAction string
+}
+
+func (e *Error) Error() string {
+	return fmt.Sprintf("an error occurred caused by: %s\nsuggestion: %s\n", e.Cause, e.SuggestedAction)
+}
+
+func NewError(cause error, suggestedAction string) error {
+	err := new(Error)
+	err.SuggestedAction = suggestedAction
+	err.Cause = cause
+
+	return err
+}
+
+func FindSuggestionForError(cause error, errorToSuggestionMap map[string]string) error {
```

Comment:
1. The name of this function can be improved. The function doesn't return "suggestion" it return a new (actionable error) or the original error. Here are some suggestions:
- `FromError()`
- `ConvertToActionable()`
- `ToActionable()`